### PR TITLE
openshifttest/ocp-logtest image update

### PR DESCRIPTION
**Pull request by LiangquanLi930**

### CODE REVIEW

File: testdata/logging/loggen/container_json_log_template.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
                         "containers": [
                             {
                                 "env": [],
-                                "image": "quay.io/openshifttest/ocp-logtest@sha256:f23bea6f669d125f2f317e3097a0a4da48e8792746db32838725b45efa6c64a4",
+                                "image": "quay.io/openshifttest/ocp-logtest:multiarch@sha256:16232868ba1143721b786dbabb3f7384645acb663fadb4af48e9ea1228a67635",
```
Comment (aleskandro) [inline testdata/logging/loggen/container_json_log_template.json:43]: the tag `:multiarch` has to be removed

File: testdata/logging/loggen/container_json_unicode_log_template.json [hunk 2]
```diff
@@ -40,7 +40,7 @@
                         "containers": [
                             {
                                 "env": [],
-                                "image": "quay.io/openshifttest/ocp-logtest@sha256:f23bea6f669d125f2f317e3097a0a4da48e8792746db32838725b45efa6c64a4",
+                                "image": "quay.io/openshifttest/ocp-logtest:multiarch@sha256:16232868ba1143721b786dbabb3f7384645acb663fadb4af48e9ea1228a67635",
```
Comment (aleskandro) [inline testdata/logging/loggen/container_json_unicode_log_template.json:43]: the tag `:multiarch` has to be removed

**Comment by aleskandro** [comment]
Refers OCPQE-6988

**Comment by LiangquanLi930** [comment]
test run: OS-20211117-0722

**Comment by aleskandro** [comment]
@QiaolingTang can you also leave a "/approve"?

**Comment by QiaolingTang** [comment]
Sorry, I didn't notice I was added as an approver.

/approve

**Pull request merged**